h2o-bindings/bin/custom/python/gen_stackedensemble.py

```diff
@@ -117,10 +130,13 @@ def extend_parms(parms):
 
 extensions = dict(
     __imports__="""
+import ast
+import json
+import warnings
+
+import h2o
```

Comment:
I know what you tried, I also try to organize imports top down:
1. core python
2. external libraries
3. current project

however, as you see below, the generated file already imports h2o modules before... so was being pragmatic here and grouped import in generated file :)

Reply:
Ok,  I will revert that to the original ordering. Should I order the imports just alphabetically or should I order them as core python alphabetically, then external alphabetically?

I know it will result in something like this:

```
from h2o.estimators.estimator_base import H2OEstimator
from h2o.exceptions import H2OValueError
from h2o.frame import H2OFrame
from h2o.utils.typechecks import assert_is_type, Enum, numeric
import ...
import ...
import ...
from h2o.utils.shared_utils import quoted
from h2o.utils.typechecks import is_type
```

Reply:
> core python alphabetically, then external alphabetically
That's what I'd like to enforce (you won't see it applied in most legacy files).
No big deal, but makes it easier to see how much a module depends from this or that.

Reply:
you know what?
can you please just fix `gen_python.py`
and mode lines 196-197 above the h2o imports?

It will modify all generated algos, but I'll be nicer.

Reply:
Ok, I guess then I should also modify the gen_*algo*.py to have sorted imports, right?

stacked ensemble imports look now like this:
```
from __future__ import absolute_import, division, print_function, unicode_literals

import ast
import json
import warnings

import h2o
from h2o.utils.shared_utils import quoted
from h2o.utils.typechecks import is_type
from h2o.estimators.estimator_base import H2OEstimator
from h2o.exceptions import H2OValueError
from h2o.frame import H2OFrame
from h2o.utils.typechecks import assert_is_type, Enum, numeric
```

Would this be ok?

Reply:
yes, better than it's ever been!